### DISCUSSION TITLE
Next.js app and missing styles

### DISCUSSION THREAD

**Discussion by BMCwebdev** [facebook/stylex#567] [Q&A] [answered]
So my other Next.js situation is settled for now. But here is the issue that kickstarted that rabbit hole.

If I start a dev server, fresh, or after killing the `./next` directory, everything looks good. But as soon as I stop the dev server, and start it back up, styles are missing. Some are there, some are not, and it isn't consistent about which one will be there or not.

I read on some discussion that the cache was an issue. I'm assuming this is what I am experiencing?

Would you be able to advise on a more targeted removal of cache items instead of the entire directory?

I see the CLI ticket has had some action ❤️

Thank you!

**Comment by BMCwebdev** [comment]
Follow up: this is happening on a Dev server, but I'm also concerned about what is happening on Prod. Will this be an issue there?

**Reply by BMCwebdev** [reply]
Dug through old disscussions and confirmed that indeed cache is an issue. I've added
```
    "predev": "rimraf .next",
    "prebuild": "rimraf .next",
```
However I'm still worried about production.

**Reply by mattstyles** [reply]
Does your prod build from scratch?
If so then it will have no cache.

**Reply by nmn** [reply, marked as answer]
We're aware of the issues integrating with Next.js and are working on a few possible paths to improve the situation.
